CHANGELOG-ABP.md

```diff
@@ -1,4 +1,10 @@
 # Release Notes
+## Chromium 69.0
+* Updated to Chromium 69.0.3497.91
```

Comment:
What about moving these changes to 68? They are also already available in 68 and will be automatically available in 69, on the other hand when we put them into 69 then it's expected that they are not available in 68.

Reply:
Let's do the following:
1. Wait for https://github.com/adblockplus/chromium/pull/10 to be approved.
2. Update 69 release notes with this bug info mentioned
3. Release Chromium 69 ( = create release tag)

After that we can cherry-pick the changes to Chromium 68 and release notes for it too.
Otherwise we can just release notes for 68 and that's it.